openlibrary/admin/templates/admin/index.html

```diff
@@ -18,8 +18,8 @@
     <h1>$_("Admin Center")</h1>
 	<div class="small">
 	    <a href="http://monitor.us.archive.org/cgi-bin/nagios3/status.cgi?hostgroup=24.openlibrary&style=detail">Nagios</a>
-	    | <a href="/admin/block">Block IPs</a>
-	    | <a href="/admin/loans">Loans</a>
+	    | <a href="/admin/block">$_("Block IPs")</a>
```

Comment:
For future work, I'd prioritize the admin pieces of the UI last since they're not exposed to the general public.